buildSrc/src/main/kotlin/cpg.frontend-dependency-conventions.gradle.kts

```diff
@@ -10,6 +10,7 @@ val enablePythonFrontend: Boolean by rootProject.extra
 val enableLLVMFrontend: Boolean by rootProject.extra
 val enableTypeScriptFrontend: Boolean by rootProject.extra
 val enableRubyFrontend: Boolean by rootProject.extra
+val enableJVMFrontend: Boolean by rootProject.extra
```

Comment:
"JVM language" seems to be a common term, see https://en.wikipedia.org/wiki/List_of_JVM_languages or https://www.baeldung.com/jvm-languages. Since this frontend might replace the "Java" one at some very distant part in the future I think the naming is ok. 

We cannot really rename the existing one to "javasource" without breaking everything.